h/templates/groups/share.html.jinja2

```diff
@@ -12,19 +12,24 @@
   <div class="content content--narrow">
     <div class="group-form is-member-of-group">
       <i class="h-icon-group group-form__heading-icon"></i>
-      <div class="group-form__heading">{{ group.name }}</div>
+      <div class="group-form__heading--short">{{ group.name }}</div>
 
       {% if document_links %}
-        Recently annotated documents:
-        <ul class="group-form-document-list">
+        <a href="{{ request.route_url('stream') }}?q=group:{{ group.pubid }}"
+           target="_blank"
+           title="Recent annotations in this group">
+           View recent group annotations</a>
+    </div>
```

Comment:
Closing an already open tag inside the `{% if document_links %}` block makes this a bit confusing to read. I'd suggest restructuring this so that tags opened and closed inside `{%if %} ... {% endif %}` paris are balanced - that might mean having two `{%if document_links %}` blocks.